docker-vertica/docker-entrypoint.sh

```diff
@@ -10,18 +10,19 @@ start_cron(){
 # because we have a Persistent Volume that backs /opt/vertica/config, so
 # it starts up empty and must be populated
 copy_config_files() {
-    # We must use sudo in case the PV was created with permissions less than 0777.
-    sudo cp -r /home/dbadmin/logrotate/* /opt/vertica/config/
+    cp -r /home/dbadmin/logrotate/* /opt/vertica/config/
+    chown -R dbadmin:verticadba /opt/vertica/config/logrotate
     rm -rf /home/dbadmin/logrotate
 
-    sudo mkdir -p /opt/vertica/config/licensing
```

Comment:
Why don't we need sudo anymore?

Reply:
We were over zealous with the sudo before and so it wasn't needed.  The long term goal is to have no reliance on sudo as that will play nice when we support OpenShift.  Removing calls to update_vertica was a key first step for that.

Reply:
Ok now I get it thanks.